tests/ttnn/github_automatic_issue.py

```diff
@@ -0,0 +1,111 @@
+# SPDX-FileCopyrightText: © 2023 Tenstorrent Inc.
+
+# SPDX-License-Identifier: Apache-2.0
+
+import os
+import hashlib
+import github
+from github import Github
+
+# GitHub
+
+# Testing repo
+"""
+REPO_OWNER = "xanderchin"
+REPO_NAME = "test_github_api"
+"""
+REPO_OWNER = "tenstorrent-metal"
+REPO_NAME = "tt-metal"
+
+# Params
+BOT_LABEL_COL = "900C3F"
+
+
+def generate_sha1_hash(input_string):
+    # Convert the input string to bytes
+    input_bytes = input_string.encode("utf-8")
+
+    # Create a new SHA-1 hash object
+    sha1_hash = hashlib.sha1()
+
+    # Update the hash object with the input bytes
+    sha1_hash.update(input_bytes)
+
+    # Get the hexadecimal representation of the hash
+    hex_hash = sha1_hash.hexdigest()
+
+    return hex_hash
+
+
+def open_repo():
+    api_key = os.environ.get("GITHUB_API_KEY")
```

Comment:
Does this work with empty `GITHUB_API_KEY`? I believe you still need a valid read key in order to access data on a org via API. I'm thinking they do this to prevent scraping